screens/Play/index.tsx

```diff
@@ -70,18 +72,38 @@ export default function PlayScreen({ route, navigation }: Props) {
 
   React.useEffect(() => {
     const loadAudio = async () => {
+      // See if file is downloaded, or to play from remote uri
```

Comment:
I like how you are commenting code here :) 

I'm a bit lost on how you figure out how the file path belongs to a particular meditation. Can you write a comment that explains how the filepaths relate to a specific meditation?